### PR TITLE
DOCSP-40815: Time series

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-40815
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40815-time-series/data-formats/time-series/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/data-formats/time-series.txt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+To learn more about the concepts mentioned in this guide, see the
+following Server manual entries:
+
+- :manual:`Time Series </core/timeseries-collections/>`
+- :manual:`Create and Query a Time Series Collection </core/timeseries/timeseries-procedures/>`
+- :manual:`Set Granularity for Time Series Data </core/timeseries/timeseries-granularity/>`
+
+To learn more about performing read operations, see :ref:`cpp-read`.
+
+To learn more about performing aggregation operations, see :ref:`cpp-aggregation`.
```
Comment (norareidy) [inline source/data-formats/time-series.txt:125]: Note to reviewer: this link will be broken until #40 is merged

File: source/data-formats/time-series.txt [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+.. important:: Server Version for Time Series Collections
+
+   To create and interact with time series collections, you must be
+   connected to a deployment running {+mdb-server+} 5.0 or later.
+
+You can create a time series collection to store time series data.  
+To create a time series collection, perform the following actions:
+
+1. Create a BSON document that specifies properties of
+   your time series collection.
```
Comment (mcmorisi) [inline source/data-formats/time-series.txt:65]: Nit

```suggestion
1. Create a BSON document that specifies the properties of
   your time series collection.
```

File: source/data-formats/time-series.txt [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+following Server manual entries:
+
+- :manual:`Time Series </core/timeseries-collections/>`
+- :manual:`Create and Query a Time Series Collection </core/timeseries/timeseries-procedures/>`
+- :manual:`Set Granularity for Time Series Data </core/timeseries/timeseries-granularity/>`
+
+To learn more about performing read operations, see :ref:`cpp-read`.
+
+To learn more about performing aggregation operations, see the :ref:`cpp-aggregation`
+guide.
```
Comment (mcmorisi) [inline source/data-formats/time-series.txt:164]: Should this be commented out as a TODO or it the aggregation page ready to go and this just needs to be merged?

**Review by mcmorisi — APPROVED** [review]
LGTM with a couple nits/questions, nice work!

File: source/data-formats/time-series.txt (hunk 3 above)
Comment (norareidy) [reply source/data-formats/time-series.txt:164]: It's ready! just need to finish the review process for it, but I'll merge aggregation before this PR

File: source/includes/data-formats/time-series.cpp [hunk 4]
```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <chrono>
+
+#include <bsoncxx/builder/list.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
```
Comment (kevinAlbs) [inline source/includes/data-formats/time-series.cpp:5]:
```suggestion
#include <bsoncxx/builder/basic/document.hpp>
```

Suggest replacing includes. This example does not use the `bsoncxx::builder::list` or `bsoncxx::builder::stream`, but does use `bsoncxx::builder::basic`.

**Review by kevinAlbs — APPROVED** [review]

**Pull request merged**